protocols/gossipsub/src/behaviour.rs

```diff
@@ -190,6 +192,7 @@ impl From<MessageAuthenticity> for PublishConfig {
                     keypair,
                     author: public_key.to_peer_id(),
                     inline_key: key,
+                    last_seq_no: rand::random(),
```

Comment:
Not sure really. I think I have some innate aversion to giving out more information about ourselves than needed. Maybe also because I was lazy about finding out which timestamp to use in go, ms vs s etc. 

Happy to change this, no real reason tbh, random just felt slightly safer, with no logical reason to back it up. :)